notebook/static/base/js/dialog.js

```diff
@@ -86,7 +86,7 @@ define(function(require) {
             var button = $("<button/>")
                 .addClass("btn btn-default btn-sm")
                 .attr("data-dismiss", "modal")
-                .text(label);
+                .text(i18n.translate(label).fetch());
```

Comment:
I see why this is in place now. I'd still like to find a way to avoid duplicating the button labels in the files where they are defined, but that needn't be solved in this PR.